libs/product/src/facades/composite-product/composite-product-facade.interface.ts

```diff
@@ -7,6 +7,9 @@ import { DaffStoreFacade } from '@daffodil/core/state';
 import { DaffCompositeProductItemOption, DaffCompositeProductItem } from '../../models/composite-product-item';
 import { DaffCompositeProduct } from '../../models/composite-product';
 
+/**
+ * A facade for accessing composite product state from an application component.
```

Comment:
The facade can be used to dispatch actions as well and its not really restricted just to components. I would say something like:
```ts
/**
 * A facade for interacting with the composite product state.
 * Exposes many parts of the state for easy access and allows dispatching of actions.
 */
```

Reply:
changed